includes/Core/User_Input/User_Input_Settings.php

```diff
@@ -0,0 +1,313 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\User_Input\User_Input_Settings
+ *
+ * @package   Google\Site_Kit\Core\User_Input
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\User_Input;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Authentication\User_Input_State;
```

Comment:
As discussed in Slack, ignoring this since `User_Input_State` will be removed as a part of [#5900](https://github.com/google/site-kit-wp/issues/5900). Thank you!